src/logging.cc

```diff
@@ -13,6 +13,7 @@ void logger_init() {
   gLoggingThreshold = boost::log::trivial::info;
   logging::add_console_log(std::cout, boost::log::keywords::format = "%Message%");
```

Comment:
It is better to add true(auto_flush) as third parameter of `add_console_log`
http://www.boost.org/doc/libs/1_54_0/libs/log/doc/html/boost/log/add_console_lo_idp21543664.html

Reply:
I think should be `logging::add_console_log(std::cout, boost::log::keywords::format = "%Message%", boost::log::keywords::auto_flush = true);`

Reply:
Oh, it is already merged (((

Reply:
I still like your solution though

Reply:
I've opened a PR for your better solution: #598